src/pages/AdminDashboard/AdminPackages.tsx

```
@@ -1,10 +1,193 @@
+import { FaEdit, FaPlus, FaTimesCircle, FaTrash } from "react-icons/fa"
+import { useState } from "react"
+import { Table } from "../../components/Table"
+import Modal from "../../components/Modal"
+import PromptsCard from "../../components/PromptsCard"
 
 const AdminPackages = () => {
+  const [showModal, setShowModal] = useState(false)
+  const [showDeleteModal, setShowDeleteModal] = useState(false)
+  const [isEditing, setIsEditing] = useState(false)
+  const [selectedPackage, setSelectedPackage] = useState<any>(null)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)